src/app/RootPrivider.tsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+
+import { Auth0Provider } from '@auth0/auth0-react';
+
+const RootProvider = ({
+  children
+}: Readonly<{
+  children: React.ReactNode;
+}>) => {
+  // Add your Auth0 domain, client ID, and redirect URI here
+  // const domain = process.env.NEXT_PUBLIC_AUTH0_DOMAIN;
+  // const clientId = process.env.NEXT_PUBLIC_AUTH0_CLIENT_ID;
+  // const redirectUri = process.env.NEXT_PUBLIC_AUTH0_REDIRECT_URI;
+  const domain = 'dev-lpnsauqdk80x5gf6.us.auth0.com';
```

Comment:
i dont think these should be here.  they should be in the .env file

Reply:
אבל אמרו לי בנתיים לשים את זה כאן כי אני לא יכול להעלות את הקובץ env ל github
כדי שזה ירוץ לבנתיים

Reply:
right sorry lets leave for now